astropy/utils/masked/function_helpers.py

```diff
@@ -110,6 +110,8 @@
     np.fix, np.isneginf, np.isposinf,
     # np.lib.function_base
     np.angle, np.i0,
+    # min and max are aliases of amin and amax:
```

Comment:
It is not necessary to add these: `np.amax is np.max` so they already are recognized.

Reply:
Got it, thanks.